.idea/codeStyles/Project.xml

```diff
@@ -161,4 +180,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```

Comment:
@secjedi These changes needs to be reverted. This was done unintentionally by you and therefore we should revert it.

Reply:
I reverted this already